tools/wpt/revlist.py

```diff
@@ -106,11 +106,14 @@ def get_parser():
                         type=int,
                         help="maximum number of revisions to be returned by "
                              "the command")
+    parser.add_argument("--verbose", action="store_true", help="debug logging")
```

Comment:
Nit: also use "-v" for convenience, which seems common practice for other commands?
```suggestion
    parser.add_argument("-v", "--verbose", action="store_true", help="debug logging")
```

Reply:
I'm not sure if we want to leave `-v` for (a hypothetical) `--version` or `--verbose`. Which other wpt commands use `-v`? Or are you talking about CLI commands in general?

Reply:
Yeah fair point - just two spots in this codebase where it's used, so likely not enough to justify making this the common pattern:
https://github.com/web-platform-tests/wpt/search?q=%22-v%22%2C+%22--verbose%22+-path%3Atools%2Fthird_party&unscoped_q=%22-v%22%2C+%22--verbose%22+-path%3Atools%2Fthird_party